Makefile

```
@@ -317,6 +317,11 @@ test-yaml:
 	@# Fall back to find if there is no git, e.g. in package builds
 	yamllint --strict $$((git ls-files "*.yml" "*.yaml" 2>/dev/null || find -name '*.y*ml') | grep -v ^dbicdh)
 
+.PHONY: test-shfmt
+test-yaml:
+	@which shfmt >/dev/null 2>&1 || (echo "Command 'shfmt' not found, can not execute bash script syntax checks" && false)
+	shfmt -d .
```

I suppose it should be named `test-yaml`
```suggestion
test-shfmt:
	@which shfmt >/dev/null 2>&1 || (echo "Command 'shfmt' not found, can not execute bash script syntax checks" && false)
	shfmt -d .
```